### PR TITLE
WIP 1.0 support

### PR DESCRIPTION
**Pull request by Keno**
Still missing a couple features.

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -35,20 +36,20 @@ println("Tuning for julia installation at $BASE_JULIA_BIN with sources possibly
 llvm_path = (Compat.Sys.isapple() && VersionNumber(Base.libllvm_version) >= v"3.8") ? "libLLVM" : "libLLVM-$(Base.libllvm_version)"
 
 llvm_lib_path = Libdl.dlpath(llvm_path)
-old_cxx_abi = searchindex(open(read, llvm_lib_path),Vector{UInt8}("_ZN4llvm3sys16getProcessTripleEv"),0) != 0
+old_cxx_abi = findfirst("_ZN4llvm3sys16getProcessTripleEv", String(open(read, llvm_lib_path))) !== nothing
```
Comment (ararslan) [inline deps/build.jl]: I think this could be simplified to
```julia
occursin("_ZN4llvm3sys16getProcessTripleEv", read(llvm_lib_path, String))
```

**Comment by ararslan** [comment]
I've attempted to help a bit in https://github.com/Keno/Cxx.jl/pull/396.

File: src/cxxtypes.jl [hunk 2]
```diff
@@ -46,7 +46,7 @@ const CxxBuiltinTs = Union{Bool, UInt8, Int8, UInt16, Int16,
 #
 # # A note on CVR qualifiers
 #
-# Though I had hoped to avoid it, correctly representing template parameters
+# Though I had hoped to aCvoid it, correctly representing template parameters
```
Comment (jekbradbury) [inline src/cxxtypes.jl]:
```suggestion
# Though I had hoped to avoid it, correctly representing template parameters
```

**Comment by Keno** [comment]
I've merged this. Still to be figured out is a new and improved build process using BinaryBuilder. Afterwards I'll tag a new version.

**Pull request closed without merging**